src/components/grids/PlanGrid.tsx

```diff
@@ -178,9 +179,89 @@ const PlanGrid = () => {
     setLayout(newLayout);
   };
 
+  const handleMove = (dx: number, dy: number) => {
```

Comment:
Could you change this to `handleItemMove`, and change `handleShiftSelect` to `handleSelectionMove`?